zebedee-reader/src/main/resources/logback.xml

```diff
@@ -23,11 +23,17 @@
         <!--</encoder>-->
     <!--</appender>-->
 
-    <!--<appender name="PRETTY_JSON" class="ch.qos.logback.core.ConsoleAppender">-->
-        <!--<encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">-->
-            <!--<layout class="com.github.onsdigital.logging.layouts.PrettyJsonLayout" />-->
-        <!--</encoder>-->
-    <!--</appender>-->
+    <appender name="PRETTY_JSON" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="com.github.onsdigital.logging.layouts.PrettyJsonLayout" />
```

Comment:
Change this to:
```
    <appender name="DP_LOGGER" class="ch.qos.logback.core.ConsoleAppender">
        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
            <layout class="com.github.onsdigital.logging.layouts.ConfigurableLayout" />
        </encoder>
    </appender>
```
Then delete the other appenders - this will function the same as it does noq but allows the logging format to be set via a env var:
`export DP_LOGGING_FORMAT=json`

Reply:
Done